### PR TITLE
CRW-1226: Do not exclude the GH Pull-Requests chapter

### PR DESCRIPTION
**Pull request by vinokurig**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Show the `managing-pull-requests` chapter in the CRW documentation.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-1226

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by yhontyk — APPROVED** [review]

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
@vinokurig Can you confirm target is Che 7.24?

**Comment by ericwill** [comment]
The docs are already in Che, this is only for CRW. As for the target version, yes it will be included for CRW 2.6.

**Comment by MichalMaler** [comment]
Please, close this issue when you will merge this:
https://issues.redhat.com/browse/RHDEVDOCS-2481
Thx @vinokurig for the PR preparation :)

Now its wait for the finishing approval from NIck @nickboldt 
https://github.com/redhat-developer/codeready-workspaces/pull/363#issuecomment-758026832

**Pull request merged**
